.dockerignore

```diff
@@ -0,0 +1,11 @@
+**/__pycache__/
+**/.mypy_cache
+**/.pytest_cache
+**/.tox
+**/.vscode
+**/.idea
+**/*.egg-info
+tests/
+docs/
+install/
+superset/assets/node_modules/
```

Comment:
Good catch for smaller docker image.

Reply:
yep, this and removing some cache makes a huge difference both in build time and image size :)